pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -194,6 +194,14 @@ func commonArgs(cluster string, cfg *config.Cluster, networkName string, nodeNam
 		args = append(args, "--volume", "/dev/mapper:/dev/mapper")
 	}
 
+	// rootless: use fuse-overlayfs by default
+	// https://github.com/kubernetes-sigs/kind/issues/2275
+	i, _ := info()
```

Comment:
this is consistent with `mountDevMapper` and `usernsRemap` that exec and parse `docker ...` commands and swallow the errors

Reply:
it would be nice if we could capture all this once but the cost is low enough that prefer getting the fix.

we should probably revisit error swallowing later ...

Reply:
TODO: should we make this `if mountFuse() {` for consistency? 🙃

Reply:
not TODO, it is less ugly 
let me change it